src/controller.rs

```diff
@@ -104,65 +168,137 @@
                 // capabilities also serve as a marker of completing initialization
                 // we park all requests from editor before initialization is complete
                 // and then dispatch them
-                if ctx.capabilities.is_some() {
+                let parked: Vec<_> = ctx
```

Comment:
something for the future: if one server is parked and another one isn't, then we can already talk to some servers